### PR TITLE
Changed cuid to a variable and added slug method to it.

### PR DESCRIPTION
**Pull request by Jnforja**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/ericelliott/cuid/blob/master/README.markdown>>
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

**Comment by Jnforja** [comment]
@ccapndave Could you review this PR ?

File: types/cuid/cuid-tests.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
-import cuid = require('cuid')
+import * as cuid from 'cuid';
```
Comment (RyanCavanaugh) [inline types/cuid/cuid-tests.ts]: Please revert - this is not a thing that works in ES6; the CJS-style import should be preferred

**Comment by RyanCavanaugh** [comment]
Merge in a few days if comment is not addressed

**Comment by Jnforja** [comment]
Changed as requested.

**Comment by ccapndave** [comment]
Sorry for the delay.  This PR looks fine to me!

**Pull request merged**